<?xml version="1.0"?>
<launch>
  <arg name="use_omni" default="false"/>
  
  <node pkg="rasberry_optimise" type="reconf_at_edges_server_new.py" name="reconf_at_edges_new" output="screen">
    <rosparam file="$(find rasberry_optimise)/resources/optim_params/optim_params_omni.yaml" command="load" if="$(arg use_omni)"/>        
    <rosparam file="$(find rasberry_optimise)/resources/optim_params/optim_params_diff.yaml" command="load" unless="$(arg use_omni)"/>
  </node>

</launch> 
